cli/examples/project/requirements.txt

```diff
@@ -1,4 +1,4 @@
 Pillow==7.1.0
 requests>=2.20.0
-numpy==1.13.3
+numpy
```

Comment:
Was this on purpose?

Reply:
The update from 1.13.3 to 1.19 was mainly because 1.13.3 was ancient (released in 2017). Then I just had it fetch latest version because in the past I very rarely have issues pairing newest numpy versions with old libraries. But.... if the the other libraries are set to specific versions, maybe it is best for consistency sake to revert back to 1.19

Reply:
yeah, and for the example project i would rather just specify an environment/version we know will work - lets revert that back to a specified version (1.19 is cool if you tested it and it works), and then we should be good to go on this PR.

Reply:
Will do. Pushing up changes in a sec

Reply:
testing 1.19 really quick again